docs/adding-new-dapps.md

```diff
@@ -17,14 +17,15 @@ There are some fields that need to be translated into different languages, and t
-| `categoryId`      | The category id under which the Dapp should show up. This will be deprecated soon, it will be replaced by `categories`.                                                                                                                                   |
-| `categories`      | An array of category ids under which the Dapp should show up.                                                                                                                                                                                             |
-| `url`             | The URL to open when the Dapp is selected. Can also be a deep link to open in-app.                                                                                                                                                                        |
-| `canPurchaseNfts` | Whether the Dapp supports purchasing NFTs. Per [App Store guidelines](https://developer.apple.com/app-store/review/guidelines/#in-app-purchase), dapps allowing to purchase NFTs won't be visible on iOS as it would require payment via in-app purchase. |
+| Property        | Description                                                                                                                                                             |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`          | The name of the Dapp.                                                                                                                                                   |
+| `id`            | A unique identifier for the Dapp. The current pattern is to use a lower-kebab-case string.                                                                              |
+| `categoryId`    | The category id under which the Dapp should show up. This will be deprecated soon, it will be replaced by `categories`.                                                 |
+| `categories`    | An array of category ids under which the Dapp should show up.                                                                                                           |
+| `url`           | The URL to open when the Dapp is selected. Can also be a deep link to open in-app.                                                                                      |
+| `listOnAndroid` | Whether to list the Dapp on Android devices. Your Dapp must be compliant with the [Play Store Content Policy](https://play.google.com/about/developer-content-policy/). |
+| `listOnIos`     | Whether to list the Dapp on iOS devices. Your Dapp must be compliant with the [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/).  |
```

Comment:
Should we keep the example of purchasing NFTs?
just to give a more concrete use case.

Reply:
I intentionally removed it: i was worried readers might focus on that one specific example and there isn't a lot of space to provide a more exhaustive answer.  we could add a FAQ or something.  E.g., "What are common issues with being listed on iOS?"

Reply:
added a faq, fyi